docs: add readmes and docs (#5)
VeryGoodOpenSource/very_good_wear_app: README.md

> # Very Good WearOS App

Reviewer: ```suggestion
# Very Good Wear OS App
```

[![Very Good Ventures][logo_white]][very_good_ventures_link_dark]
[![Very Good Ventures][logo_black]][very_good_ventures_link_light]

Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄

![coverage][coverage_badge]
[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
[![License: MIT][license_badge]][license_link]

A Very Good Flutter starter project for Wear OS devices created by the [Very Good Ventures Team][very_good_ventures_link].

> It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.

Reviewer: ```suggestion
It includes the basic initial setup and some Wear OS specific features to demonstrate how to take advantage of hardware resources.
```
sry one more

> A Very Good Flutter Starter Project for WearOS devices created by the [Very Good Ventures Team][very_good_ventures_link].
> 
> It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.
> 
> Generated by the [Very Good CLI][very_good_cli_link] 🤖

Reviewer: ```suggestion
A Very Good Flutter starter project for Wear OS devices created by the [Very Good Ventures Team][very_good_ventures_link].

It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.

Generated by the [Very Good CLI][very_good_cli_link] 🤖
```

Reviewer: It seems WearOS should be Wear OS (https://en.wikipedia.org/wiki/Wear_OS), so fixed it throughout

--- 

## Getting Started 🚀

> . Install [mason][mason_link]

Reviewer 2: ```suggestion
1. Install [mason][mason_link]
```

`dart pub global activate mason_cli`

2. Add the brick

   `mason add -g very_good_wear_app`

> 3. Make a new WearOS app

Reviewer: ```suggestion
3. Make a new Wear OS app
```

   `mason make very_good_wear_app`

> 

Reviewer 2: Should we include or point to instructions on how to run the app in a real or emulated WearOs device?

Author: We will include a link to the blogpost when we have one.

> 

Reviewer 2: Should we include instructions on how to edit the app icon (it tends to be frequently be asked)?

Author: That is a fair request that also transcend to very good core. But we don't need to solve all the problems in the first go.
